<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmHelp</class>
 <widget class="QDialog" name="frmHelp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Documentation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QSplitter" name="splitMain">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>11</number>
     </property>
     <widget class="QTabWidget" name="tabsMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabContents">
       <attribute name="title">
        <string>Contents</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <widget class="QWidget" name="tabSearch">
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
     <widget class="HelpBrowser" name="txtBrowser"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HelpBrowser</class>
   <extends>QTextBrowser</extends>
   <header>help/helpbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
